webhook-subscription-2022.md

```diff
@@ -1,22 +1,30 @@
-# Solid Webhook Notifications
+# WebHookSubscription2022
 
-Status: Proposal
+## Abstract
+The [Solid Notification Protocol](https://solid.github.io/notifications/protocol) defines a set of interaction patterns for agents to establish subscriptions to resources in a Solid Storage.
 
-## Introduction
+This specification defines a subscription type that applies these patterns to WebHooks.
 
-The Solid Notifications Specification allows for many methods by which a client is notified of actions being performed on a resource. One such method of notification is webhooks. Webhooks allow servers to register a target URL with a Solid Pod. When that server needs to be notified of an action, that target URL will be called by the Solid Pod.
+## Status of This Document
+TODO
+
+## 1. Introduction
+_This section is non-normative._
+
+The [Solid Notifications Protocol](https://solid.github.io/notifications/protocol) allows for many methods by which a client is notified of actions being performed on a resource. One such method of notification is webhooks. Webhooks allow servers to register a target URL with a Solid Pod. When that server needs to be notified of an action, that target URL will be called by the Solid Pod.
 
 Webhooks are an alternative to websockets that are more useful for server-based use cases. While a websocket solution would require a server to maintain a constant websocket connection with a Pod as long as it wants notifications, webhooks only require one registration.
 
-This document proposes an implementation for the `WebHookSubscription2021` Solid Notification protocol.
+This document proposes an implementation for the `WebHookSubscription2022` Solid Notification protocol.
 
-## Use Cases
+### 1.1. Specification Goals:
```

Comment:
Should these move to the [use case document](https://solid.github.io/notifications-panel/notifications-ucr.html)?

Reply:
Didn't know that document existed. I'm fine with it moving, but does it make sense when these use cases are webhook-specific?

Reply:
They can be a nice addition to the UCR document. I'm not interpreting those use cases as WebHook-specific. Besides... use cases shouldn't be solution-centric!

Reply:
I think this can be handled in a separate PR, especially the UCR document is in another repo. Also, anyone can pick this as a task. This shouldn't slow down the merging of this PR.